fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/rdf/JcrRdfToolsTest.java

```diff
@@ -349,37 +363,14 @@ public void shouldCreateHashUriSubjects() throws RepositoryException {
         when(mockChildNode.isNew()).thenReturn(false);
         when(testObj.jcrTools.findOrCreateNode(mockSession, "/some/#/abc", NT_FOLDER)).thenReturn(mockNode);
         when(mockHashNode.isNew()).thenReturn(true);
-        final Statement statement = testObj.skolemize(testSubjects, x);
+        final Statement statement = testObj.skolemize(testSubjects, x, "/some/#/abc");
         assertEquals(x, statement);
         verify(testObj.jcrTools).findOrCreateNode(mockSession, "/some/#/abc", NT_FOLDER);
         verify(mockNode).addMixin(FEDORA_RESOURCE);
         verify(mockHashNode).addMixin(FEDORA_PAIRTREE);
     }
 
     @Test
-    public void shouldAddBlankNodePairtreeMixin() throws RepositoryException {
-        final Model m = createDefaultModel();
-        final Resource resource = createResource();
-        final Statement x = m.createStatement(resource,
-                createProperty("info:x"),
-                resource);
-        testObj.jcrTools = mock(JcrTools.class);
-        when(testObj.jcrTools.findOrCreateNode(eq(mockSession), anyString())).thenReturn(mockNode);
-        when(mockNode.getPath()).thenReturn("/x");
-        when(mockNode.getParent()).thenReturn(mockHashNode);
-        when(mockHashNode.getParent()).thenReturn(mockChildNode);
-        when(mockHashNode.isNew()).thenReturn(true);
-        when(FedoraTypesUtils.getClosestExistingAncestor(mockSession,"/.well-known/genid/"))
-                .thenReturn(mockChildNode);
-        final Statement statement = testObj.skolemize(testSubjects, x);
-        assertEquals("info:fedora/x", statement.getSubject().toString());
-        assertEquals("info:fedora/x", statement.getObject().toString());
-        verify(testObj.jcrTools).findOrCreateNode(mockSession, "/.well-known/genid/");
-        verify(mockNode).addMixin(FEDORA_SKOLEM);
-        verify(mockNode.getParent()).addMixin(FEDORA_PAIRTREE);
-    }
```

Comment:
This test seemed redundant to me, so I just moved the `FEDORA_PAIRTREE` check to the `shouldSkolemizeBlankNodeObjects()` test above.